### ISSUE TITLE
Material icons : Two tone and other options

### ISSUE THREAD

**Issue by meetdilip** [Xaviju/inkscape-open-symbols#117]
Hi, I hope you already know that Google Material icons are now available in the following variants. 

1. Filled
2. Outlined
3. Rounded
4. Two Toned
5. Sharp

https://material.io/resources/icons/?style=baseline

It would be great if we can have them as a part of Open Symbols. Thanks

**Comment by meetdilip** [comment]
There was a set of SVG sprites which were named " symbols ". I couldn't use it even after adding them to the usr/share/inkscape/symbols directory. They were shown as " unnamed " and no matter we select any from them, it showed the same " battery " icon set. Attaching the same.

[sprites.zip](https://github.com/Xaviju/inkscape-open-symbols/files/4143485/sprites.zip)

**Comment by Xaviju** [comment]
Currently there is a Material Design Set `https://github.com/Xaviju/inkscape-open-symbols/tree/master/material-design`.
You should be able to just copy any of the svgs in that folder into the inkscape symbols folder and use it.

If you are updating symbols from Material Design, it would be nice if you put them under the same folder an reorganize it. Thanks!

**Comment by meetdilip** [comment]
> If you are updating symbols from Material Design, it would be nice if you put them under the same folder an reorganize it. Thanks!

Yes. I am trying to add other styles. I am already using the one that is available.

**Comment by meetdilip** [comment]
@Xaviju Can we discuss here ( if ok with you ).

**Comment by PanderMusubi** [comment]
@Xaviju can we move this one forward? I can also help out.

**Comment by Xaviju** [comment]
I think it basically requires to update all current Material Design Libraries in this repo to ensure that all SVG symbols have a fill and stroke value. Currently, it only have a fill value.
Its a lot of work and I cannot commit into this now but if @PanderMusubi or @meetdilip have more time go for it and I'l lbe happy to help.

**Comment by Xaviju** [comment]
My recommendation would be to check if material design has already a SVG sprite so can be copypasted. Other option is to download every icon and create the sprite using [icomoon](https://icomoon.io/) and just edit the final SVG sprite getting the missing partes from the SVG template we have in the repo

**Comment by PanderMusubi** [comment]
How can `material-design-icons/sprites/svg-sprite/svg-sprite-action.svg` be converted to `inkscape-open-symbols/material-design/material-design-action.svg`? Do you have a script for this?

I prefer first to upgrade the existing filled style and add other styles after that.

**Comment by PanderMusubi** [comment]
sort of related, a thumbs up for https://github.com/google/material-design-icons/issues/1009

**Comment by PanderMusubi** [comment]
Some more thumbs up which are important for these icons:
- https://github.com/google/material-design-icons/issues/901
- https://github.com/google/material-design-icons/issues/786
- https://github.com/google/material-design-icons/issues/743

**Comment by PanderMusubi** [comment]
See https://github.com/PanderMusubi/inkscape-open-symbols/commit/9494b3ce771baa9ce6032af09a542d148718c4bd

@yashpalgoyal1304 @qwazix This is not final. See line 17 and 18 in 2-update-material-design.sh

Does one of you know what the correct settings are, or does this differ per icon or icon set?

**Comment by yashpalgoyal1304** [comment]
> See line 17 and 18 in 2-update-material-design.sh

link??

**Comment by PanderMusubi** [comment]
https://github.com/PanderMusubi/inkscape-open-symbols/blob/master/2-update-material-design.sh#L24

**Comment by yashpalgoyal1304** [comment]
i am afraid i dont have any knowledge at all about this stuff.

**Comment by PanderMusubi** [comment]
No worries. we will find somone.

Perhaps this is one for you? https://github.com/PanderMusubi/inkscape-open-symbols/issues/135
